cloudbuild.yaml

```diff
@@ -144,21 +160,21 @@ steps:
     - FHIR_ENDPOINT=http://openmrs:8080/openmrs/ws/fhir2/R4
 
 - name: '${_REPOSITORY}/batch-pipeline:${_TAG}'
-  id: 'Run Batch Pipeline NON_JDBC'
+  id: 'Run Batch Pipeline NON_JDBC with OpenMRS source and HAPI sink'
```

Comment:
Suggestion: How about replacing `NON_JDBC` with `FHIR-search` everywhere in this file? I think that is more descriptive.

Reply:
Done